doc/routinator.1

```diff
@@ -385,18 +385,33 @@ below.
 .BR -f ,\ \fB --force
 Forces installation of the TALs even if the TAL directory already exists.
 .TP
+.B --rir-tals
+Selects the production TALs of the five RIRs for installation. If no other
+TAL selection options are provided, this option is assumed.
+.TP
+.B --rir-test-tals
+Selects the bundled TALs for RIR testbeds for installation.
+.TP
+.BI --tal= name
+Selects the bundled TAL with the provided name for installation.
+.TP
+.BI --skip-tal= name
+Deselects the bundled TAL with the given name.
+.TP
+.BI --list-tals
+List all bundled TALs and exit. The list also shows which TALs are selected
+by the
+.B --rir-tals
+and
+.B --rir-test-tals
+options.
+.TP
 .B --accept-arin-rpa
 Before you can use the ARIN TAL, you need to agree to the ARIN Relying Party
 Agreement (RPA). You can find it at
 .I https://www.arin.net/resources/manage/rpki/rpa.pdf
 and explicitly agree to it via this option. This explicit agreement is
 necessary in order to install the ARIN TAL.
-.TP
-.B --decline-arin-rpa
```

Comment:
I have brought back `--decline-arin-rpa` as an alternative to `--skip-tal arin`. It will print a warning. I decided to not document it in the man page anymore.